binding/multipart_form_mapping.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2019 Gin Core Team.  All rights reserved.
+// Use of this source code is governed by a MIT style
+// license that can be found in the LICENSE file.
+
+package binding
+
+import (
+	"errors"
+	"mime/multipart"
+	"net/http"
+	"reflect"
+)
+
+type multipartRequestSetter http.Request
```

Comment:
usually, `...er` or `...or` is interface.

Reply:
You are right! Usually..., but not always, example: [nopCloser](https://golang.org/src/io/ioutil/ioutil.go?s=3278:3299#L110)

In my case `multipartRequestSetter` is a wrapper for the `http.Request` over the `setter` interface, and previous name (`multipartRequest`) said nothing about why it wrapped.

I know about a naming of interfaces, but in this case I don't know a better name. Maybe you can suggest something for that struct?

Reply:
thanks, but I think `multipartRequest` is better. and waiting @appleboy review the pull request.